charts/pulsar/values.yaml

```diff
@@ -1151,3 +1157,28 @@ job:
   ttl:
     enabled: false
     secondsAfterFinished: 3600
+
+# This section is intended for cluster providers where all containers, including init containers,
+# need to provide the number of resources they are going to use.
+initContainer_resources:
+  zookeeper_ready:
+    requests:
+      memory: 128Mi
+      cpu: 0.1
+  cs_ready:
+    requests:
+      memory: 128Mi
+      cpu: 0.1
+  verify_cluster_id:
+    requests:
+      memory: 128Mi
+      cpu: 0.1
+  broker_ready:
+    requests:
+      memory: 128Mi
+      cpu: 0.1
+  bookkeeper_ready:
+    requests:
+      memory: 128Mi
+      cpu: 0.1
```

Comment:
done

Reply:
Since it's `requests` and not `limits`, it might not be that critical. I checked that the defaults for requests for memory in the chart for other containers are not adding any head room over `-Xmx`.